e2e/utils/random-email-generator.ts

```diff
@@ -1,4 +1,7 @@
-export const EMAIL_ADDRESS = generateAndGetEmailAddress();
+import fs from "fs";
+import path from "path";
+
+export const EMAIL_ADDRESS = getGeneratedEmail();
 export const PASSWORD = "password123";
```

Comment:
Do you have any ideas how we can configure password not to be hardcoded in the file?

Reply:
@max-teampro , there was couple options, to generate password randomly as well and save to temp file as with email. This would fit this solution the most, because we create new user before executing tests over it, then do the clean up after tests. Hence user will not exist after test run finish. Also if there would be some database, we could store sensitive data there or use envars from GitHub, which will be used to run tests on CI